### PR TITLE
fix #39672 pornl.com

### LINKED ISSUE #39672: pornl.com
### Issue URL (Annoyance)
[https://pornl.com/?source=784232601&src_hostname=PMA33GO7&c=613](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fpornl.com%2F%3Fsource%3D784232601%26src_hostname%3DPMA33GO7%26c%3D613)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Firefox
Filters: | Filter unblocking search ads and self-promotion,<br>AdGuard Base,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/39672

### CODE REVIEW

File: AnnoyancesFilter/sections/push-notifications.txt [hunk 1]
```diff
@@ -134,6 +134,7 @@ macwelt.de,pcwelt.de#%#AG_setConstant('firebase.messaging', 'noopFunc');
 ||torrentmegafilme.net/wp-content/plugins/smio-push-notification/js/frontend_webpush.js
 uol.com.br##.webalert_container
 lanacion.com.ar##.notificacion-modal
+||sw.wpu.sh/script/main.js^$domain=pornl.com
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/push-notifications.txt]: Is not fixing notifications for me:

||pornl.com/ps/index.js

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by BlazDT — CHANGES_REQUESTED** [review]
And popunder on hclips.com

**Review by BlazDT — APPROVED** [review]

**Comment by BlazDT** [comment]
And for hclips?

**Comment by zloyden** [comment]
I just found a script which caused redirect. I will make a commit.

**Comment by BlazDT** [comment]
Missed notification for hclips:
||sw.wpu.sh/script/main.js$domain=hclips.com

**Comment by zloyden** [comment]
I also checked notification for hclips before made a commit - not reproduced, but I added it.

**Review by BlazDT — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**